Lib/pydoc_data/topics.py

```diff
@@ -417,33 +417,43 @@
                'caused a\n'
                'syntax error.\n',
  'assignment-expressions': 'Assignment expressions\n'
-          '**********************\n'
-          '\n'
-          'An assignment expression (sometimes also called a “named expression”'
-          '\nor “walrus”) assigns an expression to an identifier, while also\n'
-          'returning the value of the expression.\n'
-          '\n'
-          'One common use case is when handling matched regular expressions:\n'
-          '\n'
-          '   if matching := pattern.search(data):\n'
-          '      do_something(matching)\n'
-          '\n'
-          'Or, when processing a file stream in chunks:\n'
-          '\n'
-          '   while chunk := file.read(9000):\n'
-          '      process(chunk)\n'
-          '\n'
-          'Assignment expressions must be surrounded by parentheses when used as\n'
-          'expression statements and when used as sub-expressions in slicing,\n'
-          'conditional, lambda, keyword-argument, and comprehension-if\n'
-          'expressions and in assert, with, and assignment statements. In all\n'
-          'other places where they can be used, parentheses are not required,\n'
-          'including in if and while statements.\n'
-          '\n'
-          'Added in version 3.8.\n'
-          'See also:\n'
-          '\n'
-          '  **PEP 572** - Assignment Expressions\n',
+                           '**********************\n'
```

Comment:
This seems more indented than the lines above -- intentional?

Reply:
Yep, the original version was incorrect! The new one is actually auto-generated.